src/popup/container.js

```diff
@@ -40,6 +41,7 @@ class PopupContainer extends Component {
             isPaused: false,
             archiveBtnDisabled: true,
             blacklistChoice: false,
+            isArchiveReady: false,
```

Comment:
Probably better to call this state `isArchiveEnabled` as it denotes both loading state and the enabled state here.